src/Microsoft.Health.Fhir.Api/Features/Binders/DictionaryExpansionConfigurationProvider.cs

```diff
@@ -75,4 +74,30 @@ private void EnumerateKeys(IEnumerable<string> keys, Dictionary<string, string>
             EnumerateKeys(innerKeys, data, keyPath);
         }
     }
+
+    private static bool TryParseDictionaryJson(string value, out Dictionary<string, string> dictionary)
+    {
+        dictionary = null;
+
+        if (string.IsNullOrWhiteSpace(value))
+        {
+            return false;
+        }
+
+        value = value.Trim();
+        if (!value.Trim().StartsWith("{", StringComparison.Ordinal) || !value.Trim().EndsWith("}", StringComparison.Ordinal))
```

Comment:
on line 87 you've already trimmed value and no longer need to do so

Reply:
Resolved.